websites/B/Battlefield Portal/dist/metadata.json

```diff
@@ -1,50 +1,51 @@
 {
-    "$schema": "https://schemas.premid.app/metadata/1.6",
-    "service": "Battlefield Portal",
-    "author": {
-      "name": "gala",
-      "id": "338947895665360898"
+  "$schema": "https://schemas.premid.app/metadata/1.6",
+  "service": "Battlefield Portal",
+  "author": {
+    "name": "gala",
+    "id": "338947895665360898"
+  },
+  "category": "games",
+  "logo": "https://i.imgur.com/kEnCDvm.png",
+  "thumbnail": "https://i.imgur.com/3WooHwa.png",
+  "version": "1.0.1",
+  "color": "#FF2C10",
+  "tags": [
+    "battlefield2042",
+    "portal",
+    "rules-editor",
+    "battlefield"
+  ],
+  "url": "portal.battlefield.com",
+  "description": {
+    "en": "Battlefield Portal is a platform where you will discover unexpected battles and change the rules of war. Take on custom-made modes or create your own unique experience using powerful builder tools.",
+    "vi_VN": "Battlefield Portal là nền tảng nơi bạn khám phá các trận đấu bất ngờ và thay đổi luật lệ của chiến tranh. Tham gia các chế độ tuỳ chọn hoặc tự xây dựng trải nghiệm của riêng mình bằng các công cụ đa năng."
+  },
+  "settings": [
+    {
+      "id": "block",
+      "title": "Show block/subroutine",
+      "value": true
     },
-    "category": "games",
-    "logo": "https://i.imgur.com/kEnCDvm.png",
-    "thumbnail": "https://i.imgur.com/3WooHwa.png",
-    "version": "1.0.0",
-    "color": "#FF2C10",
-    "tags": [
-      "battlefield2042",
-      "portal",
-      "rules-editor",
-      "battlefield"
-    ],
-    "url": "portal.battlefield.com",
-    "description": {
-      "en": "Battlefield Portal is a platform where you will discover unexpected battles and change the rules of war. Take on custom-made modes or create your own unique experience using powerful builder tools."
+    {
+      "id": "time",
+      "title": "Show elapsed time",
+      "value": true
     },
-    "settings": [
-      {
-        "id": "block",
-        "title": "Show block/subroutine",
-        "value": true
-      },
-      {
-        "id": "time",
-        "title": "Show elapsed time",
-        "value": true
-      },
-      {
-        "id": "buttons",
-        "title": "Show buttons",
-        "value": true      
-      },
-      {
-        "id": "name",
-        "title": "Show title",
-        "value": true
-      },
-      {
-        "id": "desc",
-        "title": "Show description",
-        "value": true
-      }
-    ]
-  }
\ No newline at end of file
+    {
+      "id": "buttons",
+      "title": "Show buttons",
+      "value": true
+    },
+    {
+      "id": "name",
+      "title": "Show title",
+      "value": true
+    },
+    {
+      "id": "desc",
+      "title": "Show description",
+      "value": true
+    }
+  ]
+}
```

Comment:
Please undo the formatting changes on this one

Reply:
Wait it's exactly the same as before though

Reply:
It’s very clearly not

Reply:
Gonna change it back first thing tomorrow morning  then, maybe VS Code is messing with me or sth

Reply:
Done(?)